apps/freeablo/faworld/spells.h

```diff
@@ -18,6 +26,20 @@ namespace FAWorld
 
         const std::string& soundEffect() const { return mSpellData.mSoundEffect; }
 
+        SpellType getType() const
+        {
```

Comment:
I would put this enum in the DiabloExe class, and change the type of DiabloExe::SpellData::mType to SpellType instead of an untyped int.